tests/integration/test_tpch.py

```diff
@@ -75,8 +76,12 @@ def test_tpch_q5(tmp_path, check_answer, get_df):
 
 
 def test_tpch_q6(tmp_path, check_answer, get_df):
+    daft.context.set_execution_config(enable_aqe=True, enable_native_executor=True)
+    start = time.time()
     daft_df = answers.q6(get_df)
     daft_pd_df = daft_df.to_pandas()
+    end = time.time()
+    print(f"Time taken: {end-start}")
```

Comment:
Currently the runtime is the same for current engine vs the simple local execution engine in this PR